src/pybind/mgr/dashboard_v2/services/__init__.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from six import add_metaclass
+
+
+class ServiceMeta(type):
```

Comment:
What's the difference between this class and `BaseControllerMeta`?

Reply:
None. I tried to come up with a more elegant solution that would avoid code duplication, but without success, so I decided to just use the same code we did before and then if someone find a more elegant solution opens a PR and changes both Service and BaseController class.

Reply:
Then let's rename `BaseControllerMeta` to `MgrContextMeta` and use that metaclass in both classes?

Reply:
ok